lab_4_summarization_textrank/main.py

```diff
@@ -2,15 +2,33 @@
 Lab 4
 Summarize text using TextRank algorithm
 """
-from typing import Union
-
+from typing import Union, Any, Type
+import re
 from lab_3_keywords_textrank.main import TextEncoder, \
     TextPreprocessor
 
 PreprocessedSentence = tuple[str, ...]
 EncodedSentence = tuple[int, ...]
 
 
+def check_type(user_input: Any, correct_type: Type) -> bool:
```

Comment:
why do you return True?  I don't understand why you need the return value if in a bad case you raise the Error

Reply:
you mean I can just return an **error** if I put **not** in a condition and that's it?

Reply:
but returning **True** helps me with going on with commands. Returning **error** allows me not to write it every time the function returns **False**